README.md

```diff
@@ -1,70 +1,118 @@
-# Getting Started with Create React App
+<!-- TABLE OF CONTENTS -->
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# 📗 Table of Contents
 
-## Available Scripts
+- [📗 Table of Contents](#table-of-contents)
+- [Todo list App -- with Webpack ](#todo-list-app--with-webpack)
+  - [🛠 Built With ](#built-with)
+    - [Key Features ](#key-features)
+  - [🚀 Live Demo ](#live-demo)
+  - [Getting Started ](#getting-started)
+    - [Install ](#install)
+  - [Author ](#author)
+  - [🤝 Contributing ](#contributing)
+  - [Show your support ](#show-your-support)
+  - [Acknowledgments ](#acknowledgments)
+  - [📝 License ](#license)
 
-In the project directory, you can run:
+<!-- PROJECT DESCRIPTION -->
 
-### `npm start`
+# Space Traveler Capstone Project <a name="todo-list-app--with-webpack"></a>
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+We build this project using React and Redux. We also consumed external API data using createAsyncThunk and extraReducers in respective redux slices.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## 🛠 Built With <a name="built-with"></a>
 
-### `npm test`
+- React
+- Redux
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+<!-- Features -->
 
-### `npm run build`
+### Key Features <a name="key-features"></a>
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+> Key features of the application include the feature to:
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- **View Rockets.**
+- **View Missions.**
+- **Reserve Rocket.**
+- **Cancel Reservation**
+- **Join Mission**
+- **Leave Mission**
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### `npm run eject`
+<!-- LIVE DEMO -->
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [optional] it is better to remove the live demo section and link since you do not have a deployed version at the moment. you can include that when you have one